crates/starknet-server/tests/common/devnet.rs

```diff
@@ -64,7 +66,14 @@ fn get_free_port() -> Result<u16, TestError> {
 impl BackgroundDevnet {
     /// Ensures the background instance spawns at a free port, checks at most `MAX_RETRIES`
     /// times
+    #[allow(dead_code)] // dead_code needed to pass clippy
     pub(crate) async fn spawn() -> Result<Self, TestError> {
+        BackgroundDevnet::spawn_with_additional_args(None).await
+    }
+
+    pub(crate) async fn spawn_with_additional_args(
+        args: Option<Vec<&str>>,
```

Comment:
This doesn't need to be Option and doesn't need to be a Vec. It would be simpler to just have `&[&str]` (slice of string references).

Then in `spawn()` implementation we would just do `spawn_with_additional_args(&[])`, no need to convert to vectors.

Reply:
Fixed in 01f57cd9350b56c38b969e2f005ebf6ee6a8e712